aiida/orm/querybuilder.py

```diff
@@ -1659,7 +1661,7 @@ def _build(self):
             isouterjoin = verticespec.get('outerjoin')
             edge_tag = verticespec['edge_tag']
 
-            if verticespec['joining_keyword'] in ('descendant_of', 'ancestor_of'):
+            if verticespec['joining_keyword'] in ('with_ancestors', 'with_descendants'):
```

Comment:
If the user uses the old 'descendant_of', will this still work? Maybe we should keep all four options for now?

Reply:
Yes, well spotted. Put these back in.